tests/e2e/watch_test.go

```diff
@@ -461,4 +465,62 @@ var _ = Describe("odoWatchE2e", func() {
 			})
 		})
 	*/
+
+	// New test template
+	//
+	//
+
+	// following command will tests in Describe section below in parallel (in 2 nodes)
+	// ginkgo -nodes=2 -focus="Example of a clean test" slowSpecThreshold=120 -randomizeAllSpecs  tests/e2e/
+	var _ = Describe("Better watch Tests", func() {
```

Comment:
this is not usefull description :-)

Reply:
what do you suggest?

Reply:
We should group all watch into one description block
@amitkrout

Reply:
`New watch tests`

Reply:
In integration tests, probably yes. It makes sense.

Reply:
why new watch test, why not add it to the existing Describe, and maybe update the name of it if you don't like it.

Reply:
this approach is good for isolation IMHO

Reply:
done @kadel, using the already present `Describe`

Reply:
Sorry for late review. 
The above comments are well directed w.r.t test. Thanks guys.